apps/api/src/report/report.service.ts

```diff
@@ -5,15 +5,16 @@ import { InjectRepository } from '@nestjs/typeorm';
 import dayjs from 'dayjs';
 import _ from 'lodash';
 
-import { IENHaPcn } from '../applicant/entity/ienhapcn.entity';
 import { DURATION_STAGES } from './constants';
-import { MilestoneDurationEntity } from './entity/milestone-duration.entity';
 import { ReportUtilService } from './report.util.service';
 import { AppLogger } from 'src/common/logger.service';
 import { IENApplicantStatus } from 'src/applicant/entity/ienapplicant-status.entity';
 import { startDateOfFiscal } from 'src/common/util';
-import { ReportPeriodDTO, StatusCategory } from '@ien/common';
+import { Authorities, ReportPeriodDTO, StatusCategory } from '@ien/common';
 import { ReportCacheEntity } from './entity/report-cache.entity';
+import { DurationEntry } from './types';
+import { DurationSummary } from './types/duration-summary';
+import { MilestoneTableEntry } from './types/milestone-table-entry';
```

Comment:
add these to `index` so we can have one import?

Reply:
combined.